modules/Link.js

```diff
@@ -55,13 +56,18 @@ class Link extends React.Component {
       isLeftClickEvent(event)
     ) {
       event.preventDefault()
-      this.context.router.transitionTo(this.props.to)
+      this.handleTransition()
```

Comment:
Not sure if this needs to be extracted here. I'd leave this out of the PR for now.